src/main/components/Widgets.js

```diff
@@ -33,18 +33,20 @@ export function getLoadingAnimationFile(){
 }
 
 export function Loading_Animation(props) {
-  
+  console.log("Rendering Loading_Animation");
```

Comment:
I was thinking we can remove this logging if you don't need it anymore.

Alternatively, we should probably just support `AppUtils.log(1, "my super detailed trace statement")` which doesn't print unless you set `AppUtils.setLogLevel(1)` for development.

The exact code for this can be copied from monero-javascript's [LibraryUtils.js.log()](https://github.com/monero-ecosystem/monero-javascript/blob/a4f1bfbf4422b8be4c1ee773008acac6ef1bb9e9/src/main/js/common/LibraryUtils.js#L20) (can be separate from this PR).